python/Ganga/Utility/files.py

```diff
@@ -12,6 +12,8 @@
 
 def expandfilename(filename, force=False):
     """expand a path or filename in a standard way so that it may contain ~ and ${VAR} strings"""
+    if not filename:
+        return filename
```

Comment:
It probably makes more sense to raise an exception that to silently return an empty string.